inference-engine/src/inference_engine/ie_compound_blob.cpp

```diff
@@ -295,22 +347,12 @@ void* CompoundBlob::getHandle() const noexcept {
     return nullptr;
 }
 
-NV12Blob::NV12Blob(const Blob::Ptr& y, const Blob::Ptr& uv) {
-    // verify data is correct
-    verifyNV12BlobInput(y, uv);
-    // set blobs
-    _blobs.emplace_back(y);
-    _blobs.emplace_back(uv);
-    tensorDesc = TensorDesc(Precision::U8, {}, Layout::NCHW);
-}
+NV12Blob::NV12Blob(const Blob::Ptr& y, const Blob::Ptr& uv)
+    : CompoundBlob({y, uv}, verifyNV12BlobInput(y, uv)) {}
 
-NV12Blob::NV12Blob(Blob::Ptr&& y, Blob::Ptr&& uv) {
-    // verify data is correct
-    verifyNV12BlobInput(y, uv);
-    // set blobs
-    _blobs.emplace_back(std::move(y));
-    _blobs.emplace_back(std::move(uv));
-    tensorDesc = TensorDesc(Precision::U8, {}, Layout::NCHW);
+NV12Blob::NV12Blob(Blob::Ptr&& y, Blob::Ptr&& uv)
+    : CompoundBlob({}, verifyNV12BlobInput(y, uv)) {
```

Comment:
NV12 blob has to have the following `TensorDesc` : `{Precision::U8, {}, Layout::NCHW}`

Reply:
done.